applications/CompressiblePotentialFlowApplication/python_scripts/compute_forces_on_nodes_process.py

```diff
@@ -29,24 +29,24 @@ def ExecuteFinalizeSolutionStep(self):
     def Execute(self):
         KratosMultiphysics.Logger.PrintInfo('ComputeForcesOnNodesProcess', 'Computing reactions on nodes')
 
-        KratosMultiphysics.VariableUtils().SetNonHistoricalVariableToZero(KratosMultiphysics.REACTION, self.body_model_part.Nodes)
+        KratosMultiphysics.VariableUtils().SetHistoricalVariableToZero(KratosMultiphysics.REACTION, self.body_model_part.Nodes)
```

Comment:
Now that everything is working, did you check whether we really need to have the reaction as a historical variable?

Reply:
If it is not set as HistoricalVariable then the solution would not converge, we will get even 0 forces in all directions